google/cloud/security/iam/dao.py

```diff
@@ -1058,18 +1066,24 @@ def add_resource_by_name(cls,
                                  session,
                                  resource_type_name,
                                  parent_type_name,
-                                 no_require_parent):
+                                 no_require_parent,
+                                 create_time=None):
             """Adds resource specified via full name."""
 
             if not no_require_parent:
                 parent = session.query(Resource).filter(
                     Resource.type_name == parent_type_name).one()
             else:
                 parent = None
-            return cls.add_resource(session, resource_type_name, parent)
+            return cls.add_resource(session, resource_type_name, parent,
+                                    create_time)
 
         @classmethod
-        def add_resource(cls, session, resource_type_name, parent=None):
+        def add_resource(cls,
+                         session,
```

Comment:
ditto: please document the parameters and the return

You can see examples here:
http://www.sphinx-doc.org/en/stable/ext/example_google.html